.env.test

```diff
@@ -13,4 +13,7 @@ APP_TOKEN=1234567890
 board_id=5ef1186ddea5ff1b03e085e6
 list_id_ToDo=5ef1186d26a8d939ea575069
 list_id_Pending=5ef1186d2fd57d026f03add0
-list_id_Done=5ef1186dcbda554f16c6d66f
\ No newline at end of file
+list_id_Done=5ef1186dcbda554f16c6d66f
+
+# MONGO DB CONNECTIONS #
+MONGODB_CONNECTION_STRING=mongodb://module11-cosmos-serverless-terraform:Cz73djsVWUvuulGi6J0AOIQhrXm1xMfoyAMAPKCCimJjDW5xsFxc55Kr4Lx8ZO0KKSNyC9eySu93OpnhzlWjSA==@module11-cosmos-serverless-terraform.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@module11-cosmos-serverless-terraform@
```

Comment:
This is a real connection string to a database that anyone can now interact with; we should remove this & rotate that password in Azure (you can do that in the portal by visiting the database -> Connection String -> select the regenerate option by your primary password)